shared/chat/conversation/messages/message-popup/exploding/container.js

```diff
@@ -25,7 +24,7 @@ const mapStateToProps = (state: TypedState, ownProps: OwnProps) => {
   const yourMessage = ownProps.message.author === state.config.username
   const meta = Constants.getMeta(state, ownProps.message.conversationIDKey)
   const _canDeleteHistory =
-    meta.teamType === 'adhoc' || TeamConstants.getCanPerform(state, meta.teamname).deleteChatHistory
+  meta.teamType === 'adhoc' || TeamConstants.getCanPerform(state, meta.teamname).deleteChatHistory
```

Comment:
same here, i understand this isn't part of this pr but this is really error prone i think

Reply:
CanPerform is per team, not per convo. Should we have a separate chat1.CanUserPerform() or something?